src/lib/parse_release.c

```diff
@@ -289,6 +289,19 @@ void parse_osinfo_for_bz(map_string_t *osinfo, char** product, char** version)
     *version = NULL;
 }
 
+void parse_osinfo_for_bug_url(map_string_t *osinfo, char** url)
+{
+    const char *os_bug_report_url = get_map_string_item_or_NULL(osinfo, "BUG_REPORT_URL");
+
+    if (os_bug_report_url)
+        *url = xstrdup(os_bug_report_url);
+    else
+        /* unset or something bad happend */
+        *url = NULL;
+
+    return;
+}
+
```

Comment:
Mostly it was an attempt to follow the behavior of the other "parse_osinfo_for" functions.  In particular it was inspired by the `parse_osinfo_for_bz` function.  Long term, I'm planning to swing back around to the other reporter plugins and add similar "default to using `BUG_REPORT_URL`" logic over time.  I was hopeful that a somewhat central function might make future patches less complex and easy to merge.

I could just in line it over in the bugzilla reporter, but this just sorta "felt right" over here since I was planning to do it again and again...